### PR TITLE
chore: Merge branch v1 into v2

### PR DESCRIPTION
**Pull request by FabianLars**

### CODE REVIEW

**Comment by FabianLars** [comment]
Alrighty, should have done that more often. Lesson learned.

File: plugins/websocket/guest-js/index.ts [hunk 1]
```diff
@@ -13,6 +13,15 @@ export interface ConnectionConfig {
   headers?: HeadersInit;
 }
 
+export interface ConnectionConfig {
```
Comment (lucasfernog) [inline plugins/websocket/guest-js/index.ts]: don't we need the change file for this change?

File: plugins/sql/src/decode/mysql.rs [hunk 2]
```diff
@@ -21,7 +21,14 @@ pub(crate) fn to_json(v: MySqlValueRef) -> Result<JsonValue, Error> {
                 JsonValue::Null
             }
         }
-        "FLOAT" | "DOUBLE" => {
+        "FLOAT" => {
```
Comment (lucasfernog) [inline plugins/sql/src/decode/mysql.rs:24]: same for this one... why is there a change file only for some of the changes?

Comment (FabianLars) [reply plugins/sql/src/decode/mysql.rs:24]: Because in the v1 branch we only added changefiles for released packages

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**